mmv1/third_party/terraform/services/storage/resource_storage_bucket_test.go.erb

```diff
@@ -76,12 +78,13 @@ func TestAccStorageBucket_basicWithAutoclass(t *testing.T) {
 				ImportStateVerify:       true,
 				ImportStateVerifyIgnore: []string{"force_destroy"},
 			},
-			// Autoclass is ForceNew, so this destroys & recreates, but does test the explicitly disabled config
 			{
-				Config: testAccStorageBucket_basicWithAutoclass(bucketName, false),
```

Comment:
What would you think about also keeping this config around to make sure that disabling works as expected?

Reply:
Good call.